lib/ansible/modules/files/lineinfile.py

```diff
@@ -125,6 +125,13 @@
     regexp: '^SELINUX='
     line: 'SELINUX=enforcing'
 
+# use double backslash to escape between double-quotes
+- name: enable IPv4 forwarding
+  lineinfile:
+    path: /etc/sysctl.conf
+    regexp: "^#net\\.ipv4\\.ip_forward=1"
```

Comment:
In order for this example to not insert a line in the file every time, make the `#` optional: `^#?net\\.ipv4\\.ip_forward=1`

Reply:
FIY the `/etc/sysctl.conf` has the `net.ipv4.ip_forward=1` commented out by default, that's the reason I've added the hashtag in the beginning. And it's idempotent, after __uncommenting__ it out it doesn't apply again.